paper/ms.tex

```diff
@@ -2018,13 +2030,16 @@ \section{Conclusions}
 G.~Laughlin, A.~Mann, and J.~Spake for clarifications and ideas, and to E.~Schaller
 and the 51 Peg science team for encouraging lightning talks at the annual summits.
 In addition, this paper relied on data collected by the TESS mission, which are publicly
-available from the Mikulski Archive for Space Telescopes (MAST).
-Funding for the TESS mission is provided by NASA’s Science Mission
-Directorate.  TIC 402980664 in particular was observed at 120-second
-cadence thanks to the TESS Guest Investigator programs G022252 (PI:
-J.~Schlieder; Sectors 18, 19, 25, 26) and G04168 (PI: R.~Jayaraman;
+available from the Mikulski Archive for Space Telescopes (MAST). Funding for the TESS 
+mission is provided by NASA’s Science Mission Directorate. TIC 402980664 in particular 
+was observed at 120-second cadence thanks to the TESS Guest Investigator programs 
+G022252 (PI: J.~Schlieder; Sectors 18, 19, 25, 26) and G04168 (PI: R.~Jayaraman;
 Sector 53).
 
+Resources supporting this work were provided by the NASA High-End Computing (HEC)
```

Comment:
Omitting, since redundant with 
"Specifically, we used the 120-second cadence light curves produced by the Science Processing and Operations Center at NASA Ames \citep{2016SPIE.9913E..3EJ}"